gui/ManageStudents.java

```diff
@@ -32,36 +41,31 @@ public ManageStudents() {
 
 	private void feedStudents() {
 		for (String student : AdminUI.roster) {
-			PacketDispatcher.sendToServer(new FeedPlayerMessage(student));
+			PacketDispatcher.sendToServer(new FeedPlayerMessage(student.split("-")[0]));
 		}
 	}
 
 	private void freezeUnfreezeStudents(boolean state) {
 		for (String student : AdminUI.roster) {
-			if (state) {
-				teacher.sendChatMessage("/p user " + student + " group add _FROZEN_");
-			} else {
-				teacher.sendChatMessage("/p user " + student + " group remove _FROZEN_");
-			}
-			PacketDispatcher.sendToServer(new RequestFreezePlayerMessage(student, state));
+			PacketDispatcher.sendToServer(new RequestFreezePlayerMessage(student.split("-")[0], state));
```

Comment:
add the commands back in since it had to be taken, its ok to have them in the admin since they have op rights